README.md

```diff
@@ -17,6 +17,11 @@ Install dependencies:
 npm install
 ```
 
+Build server: 
+```shell
+npm run build
+```
+
```

Comment:
@gurinderCodeRabbitai remember we need to add clear instructions in read me on how to start server along with all the environment variables required.

Reply:
@gurinderCodeRabbitai remember we need to add clear instructions in read me on how to start server along with all the environment variables required.

Reply:
Add this in the learning as it will be done in the future.

Reply:
@gurindercoderabbitai add this in future milestone to add variables required to start the server in readme.md

Reply:
@gurindercoderabbitai add this in future milestone to add variables required to start the server in readme.md

Reply:
@coderabbitai add this in future milestone to add variables required to start the server in readme.md

Reply:
@gurinderCodeRabbitai , this improvement of adding environment variables needed to start server in readme will be done in future. Add it in learning as a future milestone.

Reply:
@gurinderCodeRabbitai , this improvement of adding environment variables needed to start server in readme will be done in future. Add it in learning as a future milestone.

Reply:
@gurinderCodeRabbitai, Delete learning related to add environment variables in readme as it is not required anymore.

Reply:
@gurinderCodeRabbitai, Delete learning related to add environment variables in readme as it is not required anymore.